src/components/user-card/UserCard.js

```diff
@@ -21,7 +21,8 @@ function UserCardComponent(props) {
         </div>
       </div>
 
-      <Link to={`${url}/${user.id}`}>Show details</Link>
+      {show && <Link to={`${url}/${user.id}`}>Show details</Link>}  {/*додав властивість show щоб керувати відображенням
```

Comment:
very nice!

Reply:
usually naming of the prop should answer 'yes' or 'no'
for example: withShowDetail or isDetailsLinkShown

Reply:
Just to give understanding for other developers that this props should have boolean value